src/sentry/static/sentry/app/views/discover/styles.tsx

```diff
@@ -62,7 +62,7 @@ export const ResultViewActions = styled('div')`
   }
 `;
 
-export const ResultViewButtons = styled(NavTabs)<NavTabs['props']>`
+export const ResultViewButtons = styled(NavTabs)`
```

Comment:
@billyvg are we going away from this preference of props for emotion styled components in TS 3.7?

Reply:
This is invalid in 3.7 - 
```
21:43:09 webpack | ERROR in /Users/billy/Dev/sentry/src/sentry/static/sentry/app/views/incidents/details/body.tsx
21:43:09 webpack | ./app/views/incidents/details/body.tsx
21:43:09 webpack | [tsl] ERROR in /Users/billy/Dev/sentry/src/sentry/static/sentry/app/views/incidents/details/body.tsx(130,39)
21:43:09 webpack |       TS2709: Cannot use namespace 'NavTabs' as a type.

```

It also: 1) does not reference any NavTabs props in the styling and 2) can infer the component's props so is not needed.

Reply:
gotcha